### DISCUSSION TITLE
How to create new measures?

### DISCUSSION THREAD

**Discussion by GameTn** [atoti/atoti#511] [Q&A] [answered]
hello guys , i'm new to atoti and i wanna add new measures so i can do more visualisations any help  ? i want to count How many employee i have and i use this : 
![image](https://user-images.githubusercontent.com/101182803/157435606-6e75c7b8-8ab1-4a7b-8696-a8786eaab8bb.png)

**Comment by bloussou** [comment, marked as answer]
Hello,
You can look at [`tt.agg.sum()`](https://docs.atoti.io/latest/lib/atoti.agg.html#atoti.agg.sum) for instance. You can have a look at the [tutorial](https://docs.atoti.io/latest/tutorial/tutorial.html) where basic measures are defined and help is given to start with atoti.
